### PR TITLE
Add support for yolov3-tinyu in YoloV8 series

### PR DESCRIPTION
**Pull request by richard-xx**
[yolov3-tinyu](https://github.com/ultralytics/ultralytics/blob/main/ultralytics/models/v3/yolov3-tinyu.yaml) has only two output layers, I changed the output_list to be based on num_branches.

### CODE REVIEW

**Comment by tersekmatija** [comment]
Hey @richard-xx,

Good catch, thanks for your contribution. Adding @HonzaCuhel to quickly review the whole PR, test, approve, and deploy.

File: yolo/export_yolov8.py [hunk 1]
```diff
@@ -59,7 +57,7 @@ def export_onnx(self):
                         training=torch.onnx.TrainingMode.EVAL,
                         do_constant_folding=True,
                         input_names=['images'],
-                        output_names=['output1_yolov6r2', 'output2_yolov6r2', 'output3_yolov6r2'],
+                        output_names=["output{i+1}_yolov6r2" for i in range(self.num_branches) ],
```
Comment (HonzaCuhel) [inline yolo/export_yolov8.py:60]: There is missing `f` in the formatted string.
@richard-xx, could you please add f before the first double quote, so it'll be like this: `output_names=[f"output{i+1}_yolov6r2" for i in range(self.num_branches) ]`?

**Review by HonzaCuhel — CHANGES_REQUESTED** [review]
@richard-xx, nice catch! Thank you very much! There is one slight change needed

**Review by HonzaCuhel — APPROVED** [review]
@richard-xx Thank you! Seems good to me now!

**Pull request merged**